common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcherConfiguration.java

```diff
@@ -0,0 +1,43 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.internal.controllers;
+
+import com.google.gson.annotations.Expose;
+import com.microsoft.identity.common.internal.request.SdkType;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@EqualsAndHashCode
+@SuperBuilder(toBuilder = true)
+/**
+ * Class for encapsulating command dispatcher configuration
+ */
+public class CommandDispatcherConfiguration {
+
+    @Expose()
+    private int maxTheadPoolInteractive;
```

Comment:
Do we want to follow out recommended naming conventions, and have this be mMaxThreadPoolInteractive or something?

Reply:
I had seen that lombok then use the same thing with the getter... so i'll check if that's the case... if so... i'll leave as is; otherwise i'll use the standard naming convention.

Reply:
Yes.  Lombok then makes the builder method have the m prefix... i'm leaving as is.

Reply:
@shoatman you can add `@Accessors(prefix = "m")` to avoid that issue.